modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/ExportTranslationBridge.es.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ExportTranslation} from 'translation-web';
+import React from 'react';
+
+export default function ({context, props}) {
+	const MyContext = React.createContext();
```

Comment:
You could rename this context from MyContext to ExportTranslationContext

Reply:
Yeah, you are right, I was just testing but I know it is a bad habit to put "test" names 🙈